lib/feed.xml

```diff
@@ -74,6 +74,11 @@
       {% if post.category %}
         <category term="{{ post.category | xml_escape }}" />
       {% endif %}
+      {% if post.categories %}
```

Comment:
We shouldn't need to wrap the `for` loop with an `if`. If `post.categories` is empty, the `for` loop will execute zero times :+1: